### DISCUSSION TITLE
Export SBOM function does not work if you are not logged in.

### DISCUSSION THREAD

**Discussion by PowerCommands** [community/community#73235] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

On the Insight/Dependancy graph page there is a button with the caption "Export SBOM" 
If I click this button as a non logged in user I will be redirected to a page that shows a bit json code with the message "An error has happend" or something like that.

This is a bit confusing for the user, either the button to Export SBOM should not be there if you must be logged in as an user to be able to use this funcionallity or the error message should be a bit clearer so that the visitor understands that he/she has to login first.

**Comment by gitstua** [comment]
Thanks for posting in the GitHub Community, @PowerCommands! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, I've gone ahead and moved it for you. Good luck!
